### DISCUSSION TITLE
Identify cached builds vs fresh build with webpack 5

### DISCUSSION THREAD

**Discussion by zakuru** [webpack/webpack#15968] [Q&A] [answered]
Hi,

Is there a way do detect if webpack 5 is building with a cache or  doing a fresh build ?

**Comment by vankop** [comment, marked as answer]
there is no log on read.. also even if you have cache it does mean that it has correct entries..
________

on fresh run/ there is log like:
```
cache invalidated ..blabla..
```

you will need to set log level `verbose`  https://webpack.js.org/configuration/other-options/#infrastructurelogging

**Reply by zakuru** [reply]
thanks a lot for quick response
